README.md

```diff
@@ -15,6 +15,11 @@ on how to do that, including how to develop and test locally and the versioning
 
 _Note: 1.28.0 and later require Gradle 7_
 
+### 1.34.3
+*Released*: TBD
```

Comment:
If you want me to go ahead with the release with this fix now, you can update the date here to the date of the merge. Otherwise, technically both the release version and date are TBD (might change depending on what else gets included with the release).

Reply:
Thank you. Merged.